src/likes/likes.service.ts

```diff
@@ -101,7 +108,8 @@ export class LikesService {
                 id
             }
         });
-        return deletedLike
+        const action = await this.actionsService.addAction('Delete',userId,'CommentLike',deletedLike.id,deletedLike)
```

Comment:
here you have hadcoded strings. Store them in variables and reuse? insted of hardcode them